packages/cli/src/ui/hooks/usePhraseCycler.ts

```diff
@@ -29,66 +36,79 @@ export const usePhraseCycler = (
   const [currentLoadingPhrase, setCurrentLoadingPhrase] = useState(
     loadingPhrases[0],
   );
+  const showShellFocusHint = useInactivityTimer(
+    isInteractiveShellWaiting,
+    lastOutputTime,
+    5000,
```

Comment:
Nit: using a constant and putting it in constants.ts might be better just so if we ever do want to change it we only need to change one spot